Code/Framework/AzNetworking/AzNetworking/Serialization/DeltaSerializer.h

```diff
@@ -75,14 +76,14 @@ namespace AzNetworking
         bool Serialize(   float& value, const char* name,    float minValue,    float maxValue) override;
         bool Serialize(  double& value, const char* name,   double minValue,   double maxValue) override;
         bool SerializeBytes(uint8_t* buffer, uint32_t bufferCapacity, bool isString, uint32_t& outSize, const char* name) override;
-        bool BeginObject(const char *name, const char* typeName) override;
-        bool EndObject(const char *name, const char* typeName) override;
-
-        const uint8_t* GetBuffer() const override;
-        uint32_t GetCapacity() const override;
-        uint32_t GetSize() const override;
-        void ClearTrackedChangesFlag() override {}
-        bool GetTrackedChangesFlag() const override { return false; }
+        AZNETWORKING_API bool BeginObject(const char* name, const char* typeName) override;
```

Comment:
Shouldn't the `Serialize` and `SerializeBytes` function above also be exported?

Reply:
At first I thought no given the actual Serialize calls are fairly internal but unit test projects are proving a good counter-argument. So yeah, I'll be exporting these as well.